CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+0.13.2 (2020-03-02)
```

Comment:
*(2020-03-05)

Reply:
thank you, fixed by 87f1e8dcfe311af229cbf81555da4c4d71e5bbc7